src/tools/tidy/src/ui_tests.rs

```diff
@@ -17,7 +17,7 @@ use ignore::Walk;
 const ENTRY_LIMIT: u32 = 901;
 // FIXME: The following limits should be reduced eventually.
 
-const ISSUES_ENTRY_LIMIT: u32 = 1667;
+const ISSUES_ENTRY_LIMIT: u32 = 1663;
```

Comment:
Why on earth do we have *both* a file that lists all these filenames, and a constant that counts how many lines this file has? That seems like just pointless extra work when one has to update/remove a bunch of old tests like in this PR...

Cc @clubby789 @albertlarsan68 (https://github.com/rust-lang/rust/pull/110241)

Reply:
the file lists the filenames starting with issue-###. this counts the dir entry of `tests/ui/issues`. i'm pretty sure they're different.

Reply:
Ah I see, thanks.